bids/analysis/tests/test_transformations.py

```diff
@@ -45,6 +62,7 @@ def test_rename(collection):
 
 
 def test_product(collection):
+    collection = collection.clone()
```

Comment:
oh -- leftovers -- will kill now

Reply:
and killed that entire commit for now anyways -- humanity is not ready for it ;)